Windows/CommonDialog/MainWindow.xaml

```diff
@@ -32,30 +35,30 @@
                 <RowDefinition Height="*"/>
             </Grid.RowDefinitions>
             <StackPanel Orientation="Vertical" Grid.Column="0" Grid.Row="0">
-                <Button Content="Open File" Click="Button_OpenFileDialog" Style="{StaticResource DialogButtonStyle}"/>
-                <Button Content="Save File" Click="Button_SaveFileDialog" Style="{StaticResource DialogButtonStyle}"/>
-                <Button Content="Open Folder" Click="Button_OpenFolderDialog" Style="{StaticResource DialogButtonStyle}"/>
+                <Button Content="Open File" Command="{Binding OpenFileCommand}" Style="{StaticResource DialogButtonStyle}"/>
```

Comment:
You could technically use `ApplicationCommands.Open` and `Save` commands, but not an issue.